xapian-bindings/csharp/Makefile.am

```diff
@@ -92,10 +92,10 @@ $(ASSEMBLY).dll: $(XAPIAN_SWIG_CS_SRCS) AssemblyInfo.cs XapianSharp.snk
 ## this is the first time we're signing, the key in the file is used and
 ## then stored in the key container.  From then on, the key in the container
 ## is used.
-	$(CSC) -unsafe -target:library -out:$(ASSEMBLY).dll \
-	    -keyfile:XapianSharp.snk -keycontainer:"Xapian Signing Key" \
-	    `for f in $(XAPIAN_SWIG_CS_SRCS); do if test -f "$$f"; then echo $$f; else echo $(srcdir)/$$f ; fi ; done` \
-	    AssemblyInfo.cs
+	TERM= $(CSC) -unsafe -target:library -out:$(ASSEMBLY).dll \
+	          -keyfile:XapianSharp.snk -keycontainer:"Xapian Signing Key" \
+	          `for f in $(XAPIAN_SWIG_CS_SRCS); do if test -f "$$f"; then echo $$f; else echo $(srcdir)/$$f ; fi ; done` \
+	          AssemblyInfo.cs
```

Comment:
The extra indent added to these lines doesn't seem to be related to the change you're actually trying to make here (working around a mono bug).

It also seems wrong - we generally indent wrapped lines in makefiles by 4 columns, not 10.